PatientMonitoringSystem/PatientMonitoringSystem/Views/BedsideSystemView.cs

```diff
@@ -30,14 +31,8 @@ private void Updater_Tick(object sender, EventArgs e)
 		private void AddButton_Click(object sender, EventArgs e)
 		{
 			var moduleName = NameEntry.Text;
-			if (ModuleCombo.SelectedIndex == -1)
-			{
-				MessageBox.Show("Please select a module type!");
-			}
-			else {
-				var strategyType = ModuleCombo.SelectedItem.ToString();
-				controller.AddModule(moduleName, strategyType);
-			}
+			ModuleType moduleType = (ModuleType)ModuleCombo.SelectedItem;
+			controller.AddModule(moduleName, moduleType);
```

Comment:
@harrytpstudent Will this blow up if the user has not selected any option from the dropdown menu?

Reply:
Test

Reply:
Looks like the control is safe from bad user input.